core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -918,6 +918,28 @@ private void checkRelSelectivity(
 
   @Test void testDistinctRowCountTable() {
     // no unique key information is available so return null
+    RelNode rel = convertSql("select * from (values "
+        + "(1, 2, 3), "
+        + "(3, 4, 5), "
+        + "(3, 4, 5), "
+        + "(8, 4, 5) "
+        + ") t(c1, c2, c3)");
+    final RelMetadataQuery mq = rel.getCluster().getMetadataQuery();
+
+    ImmutableBitSet groupKey = ImmutableBitSet.of(0, 1, 2);
```

Comment:
Could you add a test case which contains null literal?

Reply:
Sure. Cases for null literals are added. Thanks for your good suggestion.